compile/container.go

```diff
@@ -93,14 +93,14 @@ func (m *MapSpec) TypeCode() wire.Type {
 }
 
 // ForEachTypeReference for MapSpec
-func (m *MapSpec) ForEachTypeReference(f func(TypeSpec) error) error {
-	if err := f(m.KeySpec); err != nil {
+func (m *MapSpec) ForEachTypeReference(f func(TypeSpec) error) (err error) {
```

Comment:
The named return is inconsistent with the style below. We rarely use named returns, but when you do, the `return` statement receives no arguments. This also appears to be a style change unrelated to the PR.

Reply:
This change was made to fix the lint error.  I'll update the style.